metricflow/cli/main.py

```diff
@@ -51,7 +51,7 @@
 from metricflow.model.model_validator import ModelValidator
 from dbt_semantic_interfaces.objects.user_configured_model import UserConfiguredModel
 from metricflow.protocols.sql_client import SqlEngine
-from metricflow.engine.utils import model_build_result_from_config, path_to_models
```

Comment:
I think this is what broke the tests. If path_to_models isn't used anymore we should update the patch calls. There are a bunch in the test_cli.py module, for example:

https://github.com/dbt-labs/metricflow/blob/main/metricflow/test/cli/test_cli.py#L64

Also, I'm 99% sure those patches are the thing that caused the GitPython issues, so deleting those patch calls might incidentally resolve that problem for us.

Reply:
That fixed it!